presto-native-execution/CMakeLists.txt

```diff
@@ -201,6 +204,10 @@ if(PRESTO_ENABLE_JWT)
   add_compile_definitions(PRESTO_ENABLE_JWT)
 endif()
 
+if(PRESTO_ENABLE_PROMETHEUS_REPORTER)
```

Comment:
Noob question: If `PRESTO_ENABLE_PROMETHEUS_REPORTER` is disabled/off, will we need prometheus dependency to compile and build the server?

Reply:
We don't need prometheus dependency if this flag is off. Presto Server still works.